instrumentation/que/opentelemetry-instrumentation-que.gemspec

```
@@ -25,14 +25,14 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
   spec.required_ruby_version = '>= 2.5.0'
 
-  spec.add_dependency 'opentelemetry-api', '~> 1.0.0.rc3'
+  spec.add_dependency 'opentelemetry-api', '~> 1.0'
   spec.add_dependency 'opentelemetry-instrumentation-base', '~> 0.18.2'
```

```suggestion
  spec.add_dependency 'opentelemetry-instrumentation-base', '~> 0.18.3'
```
I think this needs to be done for all instrumentation gemspecs, since we bumped the `-base` version in this PR.